buildScript/webpack.config.js

```diff
@@ -106,54 +107,88 @@ export default function makeWebpackConfig(config) {
-                    dead_code : true
+            // if using the latest uglifyjs-webpack-plugin, based on UglifyJSv3, understanding es6 modules
+            // this uglifier also minimizes the code, no LoaderOptionsPlugin is needed
+            new UglifyJsPlugin({
+                uglifyOptions: {
+                    warnings: false, // false is default
+                    compress: {
+                        ecma: 6, // 5 is default
+                        dead_code: true, // true is default
+                        unused: true,    // true is default
+                    },
+                    output: {
+                        ecma: 6
+                    }
                 }
             })
+
+            // included with webpack3 is UglifyJSv2
+            // https://github.com/webpack-contrib/uglifyjs-webpack-plugin/tree/version-0.4
+            // new webpack.optimize.UglifyJsPlugin({
+            //     sourceMap : false,
+            //     compress : {
+            //         warnings  : false,
+            //         unused    : true,
+            //         dead_code : true
+            //     }
+            // }),
+            // new webpack.LoaderOptionsPlugin({
+            //     minimize: true
+            // })
         );
     }
 
+
     /*------------------------ MODULE -----------------------------*/
-    var loaders = [
-                    {   test : /\.(js|jsx)$/,
-                        include: [config.src, config.firefly_dir,
-                                 `${config.firefly_root}/node_modules/react-component-resizable/`],
-                        loader: 'babel',
-                        query: {
-                            presets: ['es2015', 'react', 'stage-2'],
-                            plugins: ['transform-runtime']
-                        }
-                    },
-                    {   test    : /\.css$/,
-                        exclude: exclude_dirs,
-                        loaders : [
-                            'style-loader',
-                            `css-loader?root=${path.resolve(config.firefly_dir, 'html')}`,
-                            'postcss-loader'
-                        ]
-                    },
-                    {   test: /\.(png|jpg|gif)$/,
-                        loader: `url-loader?root=${path.resolve(config.firefly_dir, 'html')}`
-                    }
-                ];
+    const rules = [
+        {   test : /\.(js|jsx)$/,
+            include: [config.src, config.firefly_dir],
+            loader: 'babel-loader',
+            query: {
+                // later presets run before earlier for each AST node
+                // use 'es2015', {modules: false}] for es5 with es6 modules
+                presets: [
+                    ['env', {
+                        targets: {
+                            browsers: ['safari >= 10', 'chrome >= 63', 'firefox >= 57', 'edge >= 15']
+                        },
+                        debug: true,
```

Comment:
We should keep debug to be `true`, at least for `!PROD`. I like seeing the supported browsers and which transforms are being chosen.